### PR TITLE
Request to merge Add to do list code

### PR DESCRIPTION
**Pull request by Kingstalux**
Added the code from the to-do list project for peer feedback review.

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+  color: rgb(95, 93, 93);
+  background-color: inherit;
+}
+
+.task {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  /* margin: 1em; */
```
Comment (Ugboaja-Uchechi) [inline src/style.css:74]: No comments in your code so as to make your code neater.

**Review by Ugboaja-Uchechi — COMMENTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 

Hi @Kingstalux 

Nice work so far on this milestone of the project  😄, but there are still some changes to do.

### Required Changes ♻️

-  _There are comments in your code that needs to be removed so as to make your code neater_

-----
You can reply to this conversation if you have any questions or need elaboration. :wave: 
Ugboaja Uchechi
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/style.css (hunk 1 above)
Comment (Kingstalux) [reply src/style.css:74]: okay, corrected that.

File: src/style.css [hunk 2]
```diff
@@ -70,8 +70,6 @@ body {
   display: flex;
```
Comment (Ugboaja-Uchechi) [inline src/style.css:70]: Approved 👍

**Review by Ugboaja-Uchechi — APPROVED** [review]

**Review by Ugboaja-Uchechi — COMMENTED** [review]
Hi @Kingstalux 

Great :muscle:  job :raised_hands:  implementing all of the requirements for this project.

All linter checks have passed.

# Status: Approved 👍 :partying_face: 
Your project has been approved.

## Congratulations :+1:  :fireworks: 

_Go ahead and merge this branch. Also, don't forget to submit the project completion form.

Good luck with your next project :mechanical_arm:  :slightly_smiling_face:_ 

Happy Hacking! :computer: 

-----
You can reply to this conversation if you have any questions or need elaboration. :wave: 
Slack: @tiredCoder
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/style.css (hunk 1 above)
Comment (Ugboaja-Uchechi) [reply src/style.css]: Please remove all comments to make your code neater

**Pull request merged**